subsys/bluetooth/host/Kconfig.gatt

```diff
@@ -138,6 +138,14 @@ config BT_GATT_READ_MULTIPLE
 	  This option enables support for the GATT Read Multiple Characteristic
 	  Values procedure.
 
+config BT_GATT_AUTO_DISCOVER_CCC
+	bool "Support to automatic discover the CCC handles of characteristics"
+	default n
```

Comment:
default is 'n', should be removed. (somebody went through the entire code-base and cleaned this up earlier).

Reply:
Roger that, removed.